app/models/support_interface/providers_filter.rb

```diff
@@ -105,6 +112,17 @@ def filter_records(providers)
         providers = providers.left_outer_joins(:provider_permissions).where(provider_users_providers: { id: nil })
       end
 
+      if applied_filters[:accredited_provider].present?
+        accrediting_provider_ids = Provider
+                                    .where("CONCAT(providers.name, ' ', providers.code) ILIKE ?", "%#{applied_filters[:accredited_provider]}%")
```

Comment:
I am guessing this is quite quick anyway because there are not millions of providers but it got me wondering whether it would be more efficient to have two OR'd conditions, one on `name`, other on `code`. Probably not a performance concern, but would it be easier to read?

Reply:
Will do!